lib/Form/Templates/Templates/Template7FixedBar.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace MailPoet\Form\Templates\Templates;
+
+use MailPoet\Form\Templates\FormTemplate;
+
+class Template7FixedBar extends FormTemplate {
+  const ID = 'template_7_fixed_bar';
+
+  public function getName(): string {
+    return 'Template 7 Fixed Bar';
+  }
+
+  public function getBody(): array {
+    return [
+      [
+        'type' => 'divider',
+        'params' => [
+          'class_name' => '',
+          'height' => '1',
+          'type' => 'spacer',
+          'style' => 'solid',
+          'divider_height' => '1',
+          'divider_width' => '100',
+          'color' => 'black',
+        ],
+        'id' => 'divider',
+        'name' => 'Divider',
+      ],
+      [
+        'type' => 'columns',
+        'body' => [
+          [
+            'type' => 'column',
+            'params' => [
+              'class_name' => '',
+              'vertical_alignment' => '',
+              'width' => '50',
+            ],
+            'body' => [
+              [
+                'type' => 'heading',
+                'id' => 'heading',
+                'params' => [
+                  'content' => _x('<img draggable="false" role="img" class="emoji" alt="🕶️" src="https://s.w.org/images/core/emoji/13.0.0/svg/1f576.svg"> <strong><span style="font-family: Abril FatFace" data-font="Abril FatFace" class="mailpoet-has-font">Relax!</span></strong>', 'Text in a web form. Keep HTML tags!', 'mailpoet'),
+                  'level' => '2',
+                  'align' => 'left',
+                  'font_size' => '44',
+                  'text_color' => '#ffffff',
+                  'line_height' => '1.2',
+                  'background_color' => '',
+                  'anchor' => '',
+                  'class_name' => '',
+                ],
+              ],
+              [
+                'type' => 'paragraph',
+                'id' => 'paragraph',
+                'params' => [
+                  'content' => _x('<span style="font-family: Cairo" data-font="Cairo" class="mailpoet-has-font">Let us do the hard work for you. Sign up to receive our latest deals directly in your inbox. We\'ll never send you spam - promise. Find out more in our <a href="https://www.mailpoet.com/privacy-notice/">Privacy Policy</a>.</span>', 'Text in a web form. Keep HTML tags!', 'mailpoet'),
```

Comment:
Two things here.
1) Should we clear the link here? I think the idea is for the user to add their own privacy policy.
2) Could we not use this syntax? `We\'ll` It is hard to search for it when you need to do a full-text search in the source code.